src/planner/select_plan_test.go

```diff
@@ -471,3 +473,52 @@ func TestSelectPlanDatabaseNotFound(t *testing.T) {
 		}
 	}
 }
+
+func TestSelectPlanGetTableInfoErr(t *testing.T) {
+	query := "select * from (select * from C) as D join B on B.a = D.a join A on D.a = A.a join (E, F) on (E.a = A.a and F.a = A.a)"
+
+	log := xlog.NewStdLog(xlog.Level(xlog.PANIC))
+	database := "sbtest"
+
+	route, cleanup := router.MockNewRouter(log)
+	defer cleanup()
+
+	err := route.AddForTest(database, router.MockTableMConfig(), router.MockTableBConfig())
+	assert.Nil(t, err)
+
+	node, err := sqlparser.Parse(query)
+	assert.Nil(t, err)
+	plan := NewSelectPlan(log, database, query, node.(*sqlparser.Select), route)
+	{
+		_, err = plan.getOneTableInfo(nil)
+		want := "unsupported: aliasTableExpr cannot be nil"
+		got := err.Error()
+		assert.Equal(t, want, got)
+	}
+	{
+		tbExpr := &sqlparser.AliasedTableExpr{}
+		expr := sqlparser.TableName{
+			Name: sqlparser.NewTableIdent("C"),
+		}
+		tbExpr.Expr = expr
+		_, err = plan.getOneTableInfo(tbExpr)
+		want := "Table 'C' doesn't exist (errno 1146) (sqlstate 42S02)"
+		got := err.Error()
+		assert.Equal(t, want, got)
+	}
+	{
+		tableInfos := make([]TableInfo, 0, 4)
+		err = plan.getJoinTableInfos(plan.node.From[0].(*sqlparser.JoinTableExpr), &tableInfos)
+		want := "unsupported: JOIN.expression"
+		got := err.Error()
+		assert.Equal(t, want, got)
+	}
+	{
+		tableInfos := make([]TableInfo, 0, 4)
+		joinExpr := plan.node.From[0].(*sqlparser.JoinTableExpr)
+		err = plan.getJoinTableInfos(joinExpr.LeftExpr.(*sqlparser.JoinTableExpr), &tableInfos)
+		want := "unsupported: subqueries.in.select"
+		got := err.Error()
+		assert.Equal(t, want, got)
+	}
+}
```

Comment:
need add more test cases in the file 
maybe add the pr: planner: add select plan test [radondb#101](https://github.com/zhyass/radon/commit/6cf14132a39658dad45316616dfa05b65c0da7d5)

Reply:
planner: add select plan test radondb#101 can pr after router: add global table syntax radondb#101 commit.

Reply:
ths test case add